Inline test fail without git on PATH

The test `test_merge_input_strategy_inline_source_conflict` fails when running tests without git available.

Output:
```python
    def test_merge_input_strategy_inline_source_conflict():
        # Conflicting cell inserts at same location as removing old cell
        local = [["local\n", "some other\n", "lines\n", "to align\n"]]
        base = [["base\n", "some other\n", "lines\n", "to align\n"]]
        remote = [["remote\n", "some other\n", "lines\n", "to align\n"]]
        # Ideal case:
        if have_git:
            expected_partial = [[
                "<<<<<<< local\n",
                "local\n",
                "=======\n",
                "remote\n",
                ">>>>>>> remote\n",
                "some other\n",
                "lines\n",
                "to align\n",
                ]]
        else:
            # Fallback is not very smart yet:
            expected_partial = [[
                "<<<<<<< local\n",
                "local\n",
                "some other\n",
                "lines\n",
                "to align\n",
                #'||||||| base\n',
                #'base\n',
                #'some other\n',
                #'lines\n',
                #'to align\n',
                "=======\n",
                "remote\n",
                "some other\n",
                "lines\n",
                "to align\n",
                ">>>>>>> remote",
                ]]
        expected_conflicts = [{
            "common_path": ("cells", 0, "source"),
            "local_diff": [
                op_addrange(0, local[0][0:1]),
                op_removerange(0, 1)
                ],
            "remote_diff": [
                op_addrange(0, remote[0][0:1]),
                op_removerange(0, 1)
                ],
            "custom_diff": [
                op_addrange(0, expected_partial[0]),
                op_removerange(0, len(base[0]))
                ],
            }]
        expected_conflicts = [{
            "common_path": ("cells", 0),
            "conflict": True,
            "action": "custom",
            "local_diff": [op_patch("source", [
                op_addrange(0, local[0][0:1]),
                op_removerange(0, 1)
                ])],
            "remote_diff": [op_patch("source", [
                op_addrange(0, remote[0][0:1]),
                op_removerange(0, 1)
                ])],
            "custom_diff": [op_replace("source", "".join(expected_partial[0]))],
            }]
        merge_args = copy.deepcopy(args)
        merge_args.merge_strategy = "use-base"
        merge_args.input_strategy = "inline"
>       _check_sources(base, local, remote, expected_partial, expected_conflicts, merge_args)

nbdime\tests\test_merge_notebooks.py:655:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
nbdime\tests\test_merge_notebooks.py:202: in _check_sources
    partial, decisions = merge_notebooks(base, local, remote, merge_args)
nbdime\merging\notebooks.py:171: in merge_notebooks
    decisions = decide_notebook_merge(base, local, remote, args)
nbdime\merging\notebooks.py:147: in decide_notebook_merge
    strategies)
nbdime\merging\generic.py:1272: in decide_merge_with_diff
    path, parent_decisions, strategies)
nbdime\merging\generic.py:1247: in _merge
    path, decisions, strategies)
nbdime\merging\generic.py:808: in _merge_dicts
    subdecisions = _merge(bv, ld.diff, rd.diff, item_path, decisions, strategies)
nbdime\merging\generic.py:1251: in _merge
    path, decisions, strategies)
nbdime\merging\generic.py:1080: in _merge_lists
    item_path, decisions, strategies)
nbdime\merging\generic.py:1247: in _merge
    path, decisions, strategies)
nbdime\merging\generic.py:808: in _merge_dicts
    subdecisions = _merge(bv, ld.diff, rd.diff, item_path, decisions, strategies)
nbdime\merging\generic.py:1255: in _merge
    path, decisions, strategies)
nbdime\merging\generic.py:1211: in _merge_strings
    path, base, local_diff, remote_diff)
nbdime\merging\generic.py:578: in resolve_strategy_inline_source
    merged, status = merge_render(base, local, remote, None)
nbdime\prettyprint.py:314: in merge_render
    return builtin_merge_render(b, l, r, strategy)
nbdime\prettyprint.py:220: in builtin_merge_render
    marker_size, include_base
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

base = ['base\n', 'some other\n', 'lines\n', 'to align\n']
local = ['local\n', 'some other\n', 'lines\n', 'to align\n\n']
remote = ['remote\n', 'some other\n', 'lines\n', 'to align\n\n']
base_title = 'base', local_title = 'local', remote_title = 'remote'
marker_size = 7, include_base = False

    def format_merge_render_lines(
            base, local, remote,
            base_title, local_title, remote_title,
            marker_size, include_base):
        sep0 = "<"*marker_size
        sep1 = "|"*marker_size
        sep2 = "="*marker_size
        sep3 = ">"*marker_size

        orig_local = local
        orig_remote = remote

        if local and local[-1].endswith('\n'):
            local[-1] = local[-1] + '\n'
        if remote and remote[-1].endswith('\n'):
            remote[-1] = remote[-1] + '\n'

        # Extract equal lines at beginning
        prelines = []
        i = 0
        n = min(len(local), len(remote))
        while i < n and local[i] == remote[i]:
            prelines.append(local[i])
            i += 1
        local = local[i:]
        remote = remote[i:]

        # Extract equal lines at end
        postlines = []
        i = len(local) - 1
        j = len(remote) - 1
>       while i >= 0 and j >= 0 and local[i] == remote[j]:
E       IndexError: list index out of range

nbdime\prettyprint.py:151: IndexError
```

--- a/nbdime/prettyprint.py
+++ b/nbdime/prettyprint.py
@@ -148,7 +148,9 @@ def format_merge_render_lines(
     postlines = []
     i = len(local) - 1
     j = len(remote) - 1
-    while i >= 0 and j >= 0 and local[i] == remote[j]:
+    while (i >= 0 and i < len(local) and
+            j >= 0 and j < len(remote) and
+            local[i] == remote[j]):
         postlines.append(local[i])
         i += 1
         j += 1

